apps/settings/js/modules/bluetooth/bluetooth_device.js

```diff
@@ -30,11 +30,15 @@ define(function(require) {
    */
   return function ctor_bluetooth_device(device) {
     var type = BtClassOfDeviceMapper.getDeviceType(device.cod);
+    var connected = device.connected || false;  // default: false
     var observableBluetoothDevice = Observable({
       name: device.name,
       paired: device.paired,
       address: device.address,
-      type: type
+      type: type,
+      connected: connected,
```

Comment:
From the code the property would be set as different types of values and which is not desired. I would suggest to rename the property to `connectionStatus`. Possible values of it are all strings: 'connecting', 'connected', and 'disconnected'.